config/jobs/kubernetes/sig-node/node-docker.yaml

```diff
@@ -30,9 +30,14 @@ periodics:
     testgrid-tab-name: docker-node-conformance
 - name: ci-docker-node-conformance-kubetest2
   interval: 2h
+  branches:
+  - master
   decorate: true
-  path_alias: k8s.io/kubernetes
   extra_refs:
+  - org: kubernetes
```

Comment:
I'm curious as to why this is needed here but not on the presubmit kubetest2 jobs?

Reply:
Presubmits are always associated with the repository they are configured against and it is automatically checked out. Periodic jobs are ideally supposed to be agnostic of any repo, hence any repo needs to be explicitly checked out if required.